docs/BasicCommunication/OnSystemRequest/index.md

```diff
@@ -17,34 +17,39 @@ An `OnSystemRequest` is a notification that can be used for a couple of differen
 
 `RequestType` defines the type of the requested data from a mobile device or the cloud. The HMI may request this data, but it's SDL's responsibility to block a `SystemRequest` in case the request intends to transfer a data type not allowed by the policy table.
 
-The HMI is informed about `requestTypes` that are allowed by policies via [OnAppPermissionChanged](../../sdl/onapppermissionchanged) and [OnAppRegistered](../onappregistered).
-
-!!! NOTE
-
-If the HMI sends `OnSystemRequest` with a request type disallowed by the policy table, SDL will ignore it.
+HMI is informed about `requestTypes` that are allowed by policies via [OnAppPermissionChanged](../../sdl/onapppermissionchanged) and [OnAppRegistered](../onappregistered).
 
+!!! NOTE   
+  1. If HMI sends `OnSystemRequest` with a request type disallowed by the policy table, SDL will ignore it.   
+  2. If HMI sends 'url' parameter within BasicCommunication.OnSystemRequest, SDL resends the 'url' parameter to mobile app via OnSystemRequest.   
+  3. If HMI(SyncP) doesn't send any URLs to SDL, it is supposed that mobile application will sent Policy Table Update data back to SDL.
 !!!
-
-!!! MUST
-
-HMI must send `OnSystemRequest` if specific data is requested from the mobile device/cloud, or binary data needs to be sent to the mobile device.
-
+!!! SyncP NOTE   
```

Comment:
@DrachenkoAnastasiia These blocks only accept `NOTE`, `MUST`, `MAY`, or `IMPORTANT`. This block does not currently render correctly if you look at the preview. 

Please use the [SDL Markdown Spec](https://github.com/smartdevicelink/sdl_markdown_spec).